Cython/Compiler/ExprNodes.py

```diff
@@ -9970,8 +9970,7 @@ def analyse_types(self, env):
 
 class FuncLocalsExprNode(DictNode):
     def __init__(self, pos, env):
-        local_vars = sorted([
-            entry.name for entry in env.entries.values() if entry.name])
+        local_vars = sorted([key for key, entry in env.entries.items() if entry.name])
```

Comment:
Why was this change needed?

Reply:
The way I am injecting the `__class__` entry into the scope (for some class `C`), it looks like: `{ "__class__": Entry(<some memory address>, name=C, type=Python object)}`. Without this change, this results in the call to `locals()` returning a `dict` with two keys: `self` and `C`. With this change, it results in the two keys `self` and `__class__`. 

I'm not sure I fully understand Entries. Would it be better to create a new one which points to the same place as `C` but has `name=__class__`?